### PR TITLE
Stabilize charts

### PR DESCRIPTION
**Pull request by chaen**
Make the chart more uniform: everything (OS, SQL DB, secrets) are generated from init jobs. The CS init job only crate the folder/repo, the content is populated by external factors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,10 +38,15 @@ When running the demo some an `values.yaml` file is created as `.demo/values.yam
...
+export MYSQL_ROOT_PASSWORD=$(kubectl get secret --namespace "default" mysql-secret -o jsonpath="{.data.mysql-root-password}" | base64 -d)
+export MYSQL_PASSWORD=$(kubectl get secret --namespace "default" mysql-secret -o jsonpath="{.data.mysql-password}" | base64 -d)
+
 # Show what will be changed by running "helm upgrade"
-helm diff upgrade diracx-demo ./diracx --values .demo/values.yaml
+helm diff upgrade diracx-demo  ./diracx --values .demo/values.yaml --set rabbitmq.auth.password=$RABBITMQ_PASSWORD  --set mysql.auth.rootPassword=$MYSQL_ROOT_PASSWORD --set mysql.auth.password=$MYSQL_PASSWORD
+
 # Actually run "helm upgrade" to apply changes
-helm upgrade diracx-demo ./diracx --values .demo/values.yaml
+helm upgrade diracx-demo ./diracx --values .demo/values.yaml --set rabbitmq.auth.password=$RABBITMQ_PASSWORD  --set mysql.auth.rootPassword=$MYSQL_ROOT_PASSWORD --set mysql.auth.password=$MYSQL_PASSWORD
```
Comment (chrisburr) [inline README.md:49]: We used to be able to avoid the need to set values extracted from the cluster to upgrade

Comment (chaen) [reply README.md:49]: Hum, I do not think I have changed this logic at all. Definitely not for `RabbitMQ`. But it is very possible I ended up in a weird state at some point, I'll check.

Comment (chaen) [reply README.md:49]: Seems like it still work for performing the actual upgrade, but `diff upgrade` complains about it.

Comment (chaen) [reply README.md:49]: Yep https://github.com/databus23/helm-diff/issues/460

Comment (chaen) [reply README.md]: I've updated the comments and the commands

**Pull request merged**